Updated Identify Tool (#7115)
qgis/QGIS-Documentation: docs/user_manual/introduction/general_tools.rst

combo boxes.
:guilabel:`Mode` defines from which layers features should be identified:

* **Current layer**: only features from the selected layers are identified.

>   Current layer can also be a group or multiple selected layers. If there is no

Reviewer: The change in the first sentence already covers the "multiple selected layers" case.

```suggestion
  If a group is selected, features from its visible layers are identified. If there is no
```
